src/stubs/blend-sdk.ts

```
@@ -0,0 +1,77 @@
+    return {
+      bTokens: "0",
+      dTokens: "0",
+      collateral: "0",
+      debt: "0",
+      healthFactor: "1.0",
+    };
+  }
+}
+
+export class PoolFactory {
+  constructor(_client: BlendClient, _address: string) {}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)